converter/internal/test_common/testing.go

```diff
@@ -96,6 +105,7 @@ func validateDiags(t *testing.T, expectedDiags []string, actualDiags diag.Diagno
 		if len(expectedDiags) > ix {
 			require.Equal(t, expectedDiags[ix], diag.String())
 		} else {
+			fmt.Printf("=== EXTRA DIAGS FOUND ===\n%s\n===========================\n", actualDiags[ix:])
```

Comment:
Prints all the diags that are left, so one can more quickly fix the expectations.